pkg/dynamicaccess/controller.go

```diff
@@ -197,15 +195,12 @@ func (c *ControllerStruct) UpdateHandler(
 			return swarm.ZeroAddress, swarm.ZeroAddress, swarm.ZeroAddress, swarm.ZeroAddress, err
 		}
 	}
+	granteesToAdd := addList
```

Comment:
What is the point of `granteesToAdd`? Why not just use `addList`?

Reply:
granteesToAdd can change due to remove(), if removelist is non-empty, then granteesToAdd = gl.Get()